ocs_ci/deployment/hub_spoke.py

```
@@ -2276,6 +2276,33 @@ def deploy_ocp(self, **kwargs) -> str:
+            cluster_path = (
+                config.ENV_DATA.get("clusters", {})
+                .get(cluster_name, {})
+                .get("hosted_cluster_path")
+            )
+            try:
+                password_path = self.download_hosted_cluster_kubeadmin_password(
+                    cluster_name, cluster_path
+                )
+                if password_path:
+                    logger.info(
+                        f"Successfully downloaded kubeadmin password to {password_path}"
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)